### PR TITLE
Add admin action to mark snippet as spam or inappropiate

### PR DESCRIPTION
**Pull request by santos22**
See https://github.com/django/djangosnippets.org/issues/74

Open to feedback!

### CODE REVIEW

File: cab/admin.py [hunk 1]
```diff
@@ -13,6 +13,21 @@ class SnippetAdmin(admin.ModelAdmin):
     date_hierarchy = 'pub_date'
     search_fields = ('author__username', 'title', 'description', 'code',)
     raw_id_fields = ('author',)
+    actions = ['mark_as_inappropiate', 'mark_as_spam']
+
+    def mark_as_inappropiate(self, request, queryset):
+        for obj in queryset:
+            if not obj.flags.all():
```
Comment (santos22) [inline cab/admin.py]: Should we consider the case where a user/admin has marked a snippet as inappropriate and it later gets marked as spam or vice versa? If so, this is not supported.

Comment (chriswedgwood) [reply cab/admin.py:20]: Hi sorry I have taken so long to get back to you , I completely missed this.Thanks for the PR. Will go through it this week 👍

Comment (chriswedgwood) [reply cab/admin.py:20]: Yes I think the intent is that multiple users can mark it as spam so we wont need the check

File: djangosnippets/settings/base.py [hunk 2]
```diff
@@ -77,6 +83,7 @@ def user_url(user):
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.contrib.flatpages.middleware.FlatpageFallbackMiddleware',
     'ratelimitbackend.middleware.RateLimitMiddleware',
+    'debug_toolbar.middleware.DebugToolbarMiddleware',
```
[the remark this answers is not in the document]
Comment (santos22) [reply djangosnippets/settings/base.py]: Not sure why that was showing up but it has been removed.

**Pull request merged**